modules/ROOT/pages/x12-edi/x12-edi-connector.adoc

```diff
@@ -8,8 +8,8 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 X12 EDI Connector Version 2.1.1
 
 The Anypoint X12 EDI Connector lets you convert X12 messages to and from DataWeave-compatible representations using lists and maps.
-X12 versions 003010, 003020, 003030, 003040, 003050, 003060, 003070, 004010, 004020, 004030, 004040, 004050, 004060, 005010, 005020, 005030, 005040, 005050, and 006020 are supported with supplied schema definitions for all standard transaction sets.
-Many HIPAA document versions are also supported, as listed separately in the Connector HIPAA reference page.
+We supply the schema definitions for all https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-x12[supported X12 standard transaction sets].
+Many HIPAA document versions are also supported, as listed separately in the https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-hipaa[Connector HIPAA reference page].
```

Comment:
This should be an xref rather than web link

Reply:
Next release for sure!